grammar.js

```diff
@@ -864,6 +911,31 @@ module.exports = grammar({
       optional(seq(',', $.output_item_list))
     ),
 
+    open_statement: $ => seq(
+      caseInsensitive('open'),
+      '(',
+      choice(
+        $.unit_identifier,
+        seq($.unit_identifier, ',', $.format_identifier),
+        seq($.unit_identifier, ',', $.format_identifier, ',', commaSep1($.keyword_argument)),
+        seq($.unit_identifier, ',', commaSep1($.keyword_argument)),
+        commaSep1($.keyword_argument)
+      ),
+      ')',
+      optional($.output_item_list)
+    ),
+
+    close_statement: $ => prec(1,
```

Comment:
The `$.call_expression` rule covers the close_statement, do we gain anything labeling it explicitly? If we'd like to keep it it should probably be changed to be `seq(caseInsensitive('close'), $.argument_list)`

Reply:
yes, my rational was the same. We have open and write statements. So I added close as well.